test/base.test.js

```diff
@@ -93,6 +94,30 @@ describe('lib.base', () => {
       });
     });
 
+    describe('.delete', () => {
+      const podName = 'pod-name';
+      const query = { pretty: true };
+      const body = {
+        apiVersion: 'v1',
+        kind: 'Pod',
+        propagationPolicy: 'Foreground'
+      };
+      beforeTesting('unit', () => {
+        nock(common.api.url)
+          .delete(`/api/v1/namespaces/${ common.currentName }/pods/${ podName }`, body)
+          .query(query)
+          .reply(200, { kind: 'Pod' });
+      });
+
+      only('unit', 'should bypass query string and body from arguments into request', done => {
```

Comment:
If you make the addition I suggest above, change this to a simple `it` to support integration tests:

```js
it('should bypass query string and body from arguments into request', done => {
```